src/js/components/modals/InstallPackageModal.js

```diff
@@ -61,7 +61,7 @@ class InstallPackageModal
         name: "cosmosPackages",
         events: [
           "descriptionError",
```

Comment:
We need to change the `descriptionError` too don't we?

Reply:
Yes, good catch